client/web/compose/src/components/ModuleFields/Configurator/Select.vue

```diff
@@ -184,6 +184,23 @@ export default {
     isNew () {
       return this.module.moduleID === NoID || this.field.fieldID === NoID
     },
+
+    selectOptions () {
+      const options = [
+        { text: this.$t('kind.select.optionType.default'), value: 'default' },
+        { text: this.$t('kind.select.optionType.multiple'), value: 'multiple' },
+        { text: this.$t('kind.select.optionType.each'), value: 'each' },
+        { text: this.$t('kind.select.optionType.list'), value: 'list' },
+      ]
+      if (!this.f.isMulti) {
+        return options.filter(o => o.value === 'list' || o.value === 'default')
+      }
+      return options
+    },
+
+    shouldAllowDuplicates () {
+      return this.f.options.selectType !== 'multiple' && this.f.isMulti && this.f.options.selectType !== 'list'
```

Comment:
Lets make this more readable, you can add a property to the option that will indicate if it supports duplicates.

```
if (!this.f.isMulti) return false

const { duplicates = false } = this.options.find(({ value }) => value === this.f.options.selectType) || {}
return duplicates 
```